docs/source/data_acquisition/AzureData.rst

```diff
@@ -53,28 +53,30 @@ The first step in using this package is to install the msticpy package.
     Imports Complete
 
 
-Instantiating and Connecting with an Azure Data Connector
+Instantiating and Connecting with an Azzre Data Connector
```

Comment:
Azzre

Reply:
Damn! That was a typo I introduced to check the spell checking functionality!